lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -32,6 +30,8 @@ function wp_theme_has_theme_json( $clear_cache = false ) {
 			$theme_has_support = is_readable( get_template_directory() . '/theme.json' );
 		}
 
+		wp_cache_set( $cache_key, $theme_has_support );
```

Comment:
I'm considering setting an expiration period. While cache invalidation should be well managed, this should serve as a last resort in case something goes hairy. The issue is: which would be a good expiration?

Reply:
Please do not add an expiration here. Let's trust that cache invalidation works. There is no need for a expiration.

Reply:
+1, we should rely on cache invalidation, not expiration for stable information like this.

Reply:
```suggestion
		wp_cache_set( $cache_key, (int) $theme_has_support, 'themes' );
```

Reply:
ok, let's try without an expiration.